server/src/test/java/io/crate/expression/operator/LikeOperatorTest.java

```diff
@@ -223,4 +223,11 @@ public void test_like_with_non_single_char_escape_throws_error() {
     public void test_like_with_empty_escape_disables_escaping() {
         assertEvaluate("'Test' LIKE 'T\\%' ESCAPE ''", false);
     }
+
+    @Test
+    public void test_like_ilike_with_trailing_escape_char_throws_error() {
+        assertThatThrownBy(() -> LikeOperators.patternToRegex("akak", 'k'))
```

Comment:
It's not obvious to me where the escape char is in this input?

Reply:
Changed the assertions to use a full LIKE and ILIKE expression.